### DISCUSSION TITLE
Clarification on "PasswordUnset" Error when logging in

### DISCUSSION THREAD

**Discussion by alvinjz2** [ValvePython/steam#386] [Q&A]
Encountering the PasswordUnset error. From the docs, it says that "Password could not be verified as it’s unset server side", but I'm still somewhat confused as to what this means. I'm just passing in a string for both my username and password, which I've verified to be correct from logging in manually onto the steam website. 

```
import os
import json
from dotenv import load_dotenv
load_dotenv()

from steam.client import SteamClient
from steam import guard

username, password = os.environ.get('username'), os.environ.get('password')
client = SteamClient()
secret = json.load(open(os.environ.get('secrets')))
SA = guard.SteamAuthenticator(secret)
result = client.login(username=username, password=password, two_factor_code=SA.get_code())
print(result)
```
This is what I'm getting as a result: 
`EResult.PasswordUnset`

Any advice would be greatly appreciated!

**Comment by rossengeorgiev** [comment]
Result code suggests that password is not set. My suggestion is to enable verbose logging and see whats in the packets. See https://steam.readthedocs.io/en/latest/user_guide.html#logging
